ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__virtualised-hub-cluster-stable.yaml

```diff
@@ -0,0 +1,68 @@
+build_root:
+  project_image:
+    dockerfile_literal: |-
```

Comment:
Maybe we can store the Dockerfile in our repo? we had [something](https://github.com/openshift-kni/eco-ci-cd/blob/main/openshift-ci/Dockerfile) in previous tests, maybe we can modify it and use it instead of having it here for readability?